### PR TITLE
Release 2.8.0

### PR DESCRIPTION
**Pull request by henrypalacios**
# Release 2.8.0

:warning::male_detective:  DO NOT SQUASH :male_detective: :warning: 

## Main features:
- Integration of the amazing **CoW protocol SDK** package [`cow-sdk`](https://github.com/cowprotocol/cow-sdk) 
- Make the Batch viewer more accessible, add button in the detail view and allow **URL anchoring** to _tab_ with tx graphic.
- Miscellaneous improvements, bug fix
    - _Batch Viewer_, Different trades are now shown when there is more than one interaction with the same source/target **AMMs**.
    - _Improved UI_, we have better positioned the page loaders and included an fantastic **CoW** custom page loader.

### Features
1c7ccea6 Adding class to edges with one more same direction (#84)
eb5873ac Merge branch 'main' into release/2.8.0-rc.0
2f7ee666 Loading improvements (#80)
8ca4258c Adding useQuery search URL params hook (#49)
d96287f8 make multiple edges to target visible in the batch viewer (#78)
8725ce77 update network box label in the batch viewer (#79)
cebd490b Merge pull request #56 from cowprotocol/1066/cow-sdk-integration
91a4d81b Use SupportedChainId type from sdk
c9ee4f1c Add type to tests
6c83307e Fix property on Order type
ddf94c58 Remove unused types
4473d5f4 Update types on tests
b5be5049 Update types and remove old tests
23c9ada8 Fix types for Order and Trade
b48f3f77 Update types
c93150ec Update repositories URL and chat support (#53)

### Dependencies Updates
8f338361 Bump to definitive version 2.8.0
7b6cfc55 Bump version to 2.8.0-rc.0
a5349593 Bump cow-sdk version

### CODE REVIEW

**Comment by elena-zh** [comment]
Hey @henrypalacios , @ramirotw , great PR!
However, I think, we should include #84 PR into it (if possible).

**Comment by elena-zh** [comment]
Hey, something wrong with SDK for Rinkeby in the PR: 
image

**Comment by ramirotw** [comment]
> Hey, something wrong with SDK for Rinkeby in the PR: image
> 
> However, seems like it was a temporary issue: now it works fine.

Yes, the Barn API doesn't have a 100% uptime and sometimes you receive a 503 http error

**Comment by henrypalacios** [comment]
> Hey @henrypalacios , @ramirotw , great PR!
> However, I think, we should include https://github.com/cowprotocol/explorer/pull/84 PR into it (if possible).


I agree, my intention was to create an interim solution to not release with the overlapping we were getting in most cases (😅, although it grew more than planned)

**Review by elena-zh — APPROVED** [review]

**Review by alfetopito — APPROVED** [review]
👍 

Only one small comment that does not block the release

File: src/services/helpers/tryGetOrderOnAllNetworks.ts [hunk 1]
```diff
@@ -1,9 +1,10 @@
 import { GetOrderParams, GetTxOrdersParams, RawOrder } from 'api/operator'
 import { NETWORK_ID_SEARCH_LIST } from 'apps/explorer/const'
 import { Network } from 'types'
+import { OrderMetaData } from '@cowprotocol/cow-sdk'
 
-export type SingleOrder = RawOrder | null
-export type MultipleOrders = RawOrder[] | null
+export type SingleOrder = RawOrder | OrderMetaData | null
+export type MultipleOrders = RawOrder[] | OrderMetaData[] | null
```
[the remark this answers is not in the document]
Comment (henrypalacios) [reply src/services/helpers/tryGetOrderOnAllNetworks.ts:7]: I assume that you mean that `OrderMetaData` and `RawOrder`.

I'll check it out and get back to you.

**Comment by henrypalacios** [comment]
@ramirotw, @matextrem could we make a temporary definition so as not to use `any` in [COW_SDK](https://github.com/cowprotocol/explorer/blob/develop/src/const.ts#L232) statement?

Or will you have the types defined in the SDK package?

**Pull request merged**